get_dependencies.py

```diff
@@ -38,10 +38,9 @@ def download(url, filename):
 download(url, name)
 
 print('[*] Extracting files...')
-f = tarfile.open(name, 'r:xz')
-f.extractall('.')
-f.close()
-
+with tarfile.open(name, 'r:xz') as f:
+    f.extractall('.')
+    f.close()
```

Comment:
This is still using tarfile.open and not the TarFile class.

Reply:
Check [this part](https://docs.python.org/3/library/tarfile.html#tarfile.TarFile) of the docs.

Reply:
This is the relevant part: https://docs.python.org/3/library/tarfile.html#tarfile-objects.

Reply:
Ah I see that `tarfile.open` can be used as a context manager too, so we can keep using it.
You don't need to close it anymore though, as it is done automatically through the context manager.

Reply:
(See https://docs.python.org/3/library/tarfile.html#tarfile.TarFile.open for reference, `tarfile.open` is a shortcut to this)

Reply:
Oh, I see. :O The recommended way is to use `tarfile.open` though, so I think we should keep that?

Reply:
They are basically the same, as `tarfile.open` is a shortcut to `TarFile.open`. Since we are already using `tarfile.open` there's no need to change to `TarFile.open`, the only thing we need to do is make use of the context manager.

Reply:
Okay got it.